crates/bevy_render/src/batching/mod.rs

```diff
@@ -56,7 +56,9 @@ impl<T: PartialEq> BatchMeta<T> {
 /// A trait to support getting data used for batching draw commands via phase
 /// items.
 pub trait GetBatchData {
+    type Param: SystemParam + 'static;
     type Query: ReadOnlyWorldQuery;
+    type QueryFilter: ReadOnlyWorldQuery;
```

Comment:
We probably need to remove the dependency on `Entity` for `PhaseItem`s in the future.

Reply:
I have big plans for PhaseItems. :)